src/main/kotlin/com/hibob/academy/dao/PetDao.kt

```diff
@@ -0,0 +1,89 @@
+package com.hibob.academy.dao
+
+import org.jooq.DSLContext
+import org.jooq.Record
+import org.jooq.RecordMapper
+import org.jooq.impl.DSL
+import org.springframework.stereotype.Component
+import java.math.BigInteger
+import java.time.LocalDateTime
+import java.sql.Date
+
+data class PetCreationRequest(
+    val name: String,
+    val type: String,
+    val dateOfArivval: Date,
+    val companyId: Long,
+    val ownerId: Long
+)
+
+data class Pet(
+    val id: Long,
+    val name: String,
+    val type: String,
+    val dateOfArivval: Date,
+    val companyId: Long,
+    val ownerId: Long
```

Comment:
Do all pets have owner?
What about pets that havent been adopted yet?

Reply:
No - so i change now - it can be nullable.